libs/client/shared/ui/view-header/src/lib/view-header/view-header.component.html

```diff
@@ -1,2 +1,9 @@
-<p>view-header works!</p>
-<pre>{{ header | json }}</pre>
+<div class="view-header__wrapper">
+  <button tuiIconButton appearance="flat" (click)="onClickPrevButton()">
+    <tui-svg src="tui::tuiIconChevronLeftLarge"></tui-svg>
+  </button>
+  <div>{{header}}</div>
```

Comment:
as this is a UI component, we probably want to show the page title that is in this model, not the entire data
![view-header](https://user-images.githubusercontent.com/23154054/213193978-25acb5e9-743c-44f4-87ed-58e9155dcde7.PNG)